ccmbioinfo/lfs
# CARDS-1475: PROMs dashboard: one "Appointments" box + 5 more boxes, one per questionnaire, with the score displayed as one of the columns

### Testing:
* Import [testPatients.csv](https://github.com/data-team-uhn/cards/files/7954455/testPatients.csv):
`curl -u admin:admin "localhost:8080/Forms/" -X POST -F ":data=@testPatients.csv" -F":questionnaire=/Questionnaires/Patient information" -F ":subjectType=/SubjectTypes/Patient"`
* Import [testVisits.csv](https://github.com/data-team-uhn/cards/files/7954456/testVisits.csv) :
`curl -u admin:admin "localhost:8080/Forms/" -X POST -F ":data=@testVisits.csv" -F":questionnaire=/Questionnaires/Visit information" -F ":subjectType=/SubjectTypes/Patient" -F ":subjectType=/SubjectTypes/Patient/Visit"`
* Log in with admin or clinician user
* There should be a link in the side bar labeled **PMCC** - click it to get to the proms-specific dashboard
* Switch to the patient UI and identify as a test patient, fill out surveys
* Go back to the dashboard to see the boxes get populated
